product-service/handlers/getProductsList.js

```diff
@@ -1,16 +1,24 @@
-import mockProducts from "../mock/products";
+import pg from "pg";
+import dbOptions from "../config/db";
+import commonResponse from "../utils/commonResponse";
 
 const getProductsList = async (event) => {
-  const headers = {
-    "Access-Control-Allow-Origin": "*",
-    "Access-Control-Allow-Credentials": true,
-  };
+  const client = new pg.Client(dbOptions);
 
-  return {
-    statusCode: 200,
-    headers,
-    body: JSON.stringify(mockProducts),
-  };
+  try {
+    console.log("getProductsList", JSON.stringify(event));
+
+    await client.connect();
+
+    const { rows: products } = await client.query(
+      "SELECT * from products JOIN stocks ON products.id = stocks.product_id"
```

Comment:
You can select only the fileds you really need. If you check current response, you'll see 2 identical fields: `id` and `product_id`